### DISCUSSION TITLE
Baseline 모델 prediction과 label plot을 공유합니다.

### DISCUSSION THREAD

**Discussion by ymnseol** [boostcampaitech4nlp2/level1_semantictextsimilarity_nlp-level1-nlp-13#10] [Show and tell]
| Base model | Batch size | Epochs | Learning rate | Loss |
|-------------|-----------|---------|--------------|------|
| klue/roberta-small | 64 | 10 | 1e-5 | MAE |

최종 validation pearson correlation coefficient는 0.8613234162330627입니다.
학습이 완료된 모델로부터 얻은 dev.csv prediction과 label 사이의 pearson corr. coef.는 0.8605입니다.

![image](https://user-images.githubusercontent.com/105059564/198047823-2c203fb8-babd-47c5-b36d-17264e3e7e56.png)

**Comment by papari1123** [comment]
오 내가할려고 했는데.. 고맙습니다.
